src/CLIMA-atmos/Parameterizations/SurfaceFluxes/src/SurfaceFluxes.jl

```diff
@@ -0,0 +1,145 @@
+"""
+    SurfaceFluxes
+
+Surface flux functions, e.g., for buoyancy flux,
+friction velocity, and exchange coefficients.
+
+"""
+module SurfaceFluxes
+
+using Utilities.RootSolvers
+using Utilities.MoistThermodynamics
+using PlanetParameters
+
+function compute_buoyancy_flux(
+  shf::R,
+  lhf::R,
+  T_b::R,
+  qt_b::R,
+  ql_b::R,
+  qi_b::R,
+  alpha0_0::R
+  )::R where R
+  cp_ = cp_m(qt_b, ql_b, qi_b)
+  lv = latent_heat_vapor(T_b)
+  temp1 = (PlanetParameters.eps_vi-1.0)
+  temp2 = (shf + temp1 * cp_ * T_b * lhf /lv)
+  return (PlanetParameters.grav * alpha0_0 / cp_ / T_b * temp2)
+end
+
+function ψ_m_unstable(ζ::R, ζ_0::R) where R
+  x(ζ) = (1.0 - PlanetParameters.gamma_m * ζ)^0.25
+  temp(ζ, ζ_0) = log((1.0 + ζ)/(1.0 + ζ_0))
```

Comment:
Watch out for possible type promotion here. Will Julia automatically keep the type of zeta? If not, please change it so it does.